build/questbookLocalisationShift.py

```diff
@@ -0,0 +1,46 @@
+# Created by The Science Demon for trainvoi
+# Next time, don't have such a skill issue, thanks
+
+import json
+import os
+
+questbookPath = os.path.normpath(os.path.abspath(f"{__file__}/../../config/betterquesting/DefaultQuests.json"))
+
+def Setup():
+	questbook = GetQuestbook()
+
+	ShiftLocalisationKeys(questbook = questbook, location = "questDatabase:9")
+
+	input("Press Enter to exit...")
+
+def GetQuestbook():
+	with open(questbookPath, "r") as file:
+		return json.load(file)
+
+def ShiftLocalisationKeys(questbook: dict, location: str):
+	for quest in dict(questbook[location]):
+		questId = questbook[location][quest]["questID:3"]
+		print(f"Found quest with ID: {questId}")
+
+		wantedQuestTitel = "susy.quest.db.%s.%s" % (questId, "title")
```

Comment:
This variable name does appear to have a spelling error in it? It's also copied throughout the rest of the code.